README.md

```diff
@@ -918,6 +918,23 @@ Options for the [OpenId Connect](http://developer.okta.com/docs/api/resources/oi
       authScheme: 'OAUTH2'
     }
     ```
+- **customButtons** - Adds an array of custom Auth buttons configurable by the properties below:
+
+    - `title` - String to set button text
+    - `className` - Sets a class on the button
+    - `dataAttr` - Sets data-se attribute on the button
```

Comment:
I don't think we want to expose this publicly - data-se is our own internal data attr for selenium testing. They'd never need to set this.